src/Symfony/Component/HtmlSanitizer/Parser/MastermindsParser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer\Parser;
+
+use Masterminds\HTML5;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @experimental
+ */
+final class MastermindsParser implements ParserInterface
+{
+    public function parse(string $html): ?\DOMNode
+    {
+        return (new HTML5())->loadHTMLFragment($html);
```

Comment:
missing constructor to allow defining the $defaultOptions argument of the HTML5 class?